homeassistant/components/homekit/type_switches.py

```diff
@@ -111,11 +110,8 @@ def __init__(self, *args):
 
     def is_activate(self, state):
         """Check if entity is activate only."""
-        can_cancel = state.attributes.get(ATTR_CAN_CANCEL)
         if self._domain == "scene":
             return True
-        if self._domain == "script" and not can_cancel:
```

Comment:
This is going to be a breaking change for Homekit I think? (breaking has already happened in the other PR)

I wonder if Homekit should just always make these "activate only"? 

CC @bdraco

Reply:
It shouldn't be breaking I guess, `can_cancel` would always be true.